File: manager-web/src/main/resources/static/js/service/contentService.js
//服务层
app.service('contentService',function($http){
	    	

	    //查找所有
        this.findAll = function () {
            return $http.get("./content");
        }
    
        //分页查找
        this.findPage = function (pageNo, pageSize) {
            return $http.get("./content/" + pageNo + "/" + pageSize);
        }
    
        //添加数据
        this.save = function (entity) {
            return $http.post("./content", entity);
        }
    
        //修改数据
        this.update = function (entity) {
            return $http.put("./content", entity);
        }
    
        //删除数据
        this.delete = function (selectIds) {
            return $http.delete("./content?ids=" + selectIds);
        }
    
        //模糊查询数据
        this.search=function (pageNo, pageSize,entity) {
            return $http.post("./content/search/" + pageNo + "/" + pageSize, entity);
        }
		//删除单行数据
        this.deleteOne = function (id) {
            return $http.delete("./content/" + id);
    }
  	
});

File: content-interface/src/main/java/com/gm/service/ContentService.java
package com.gm.service;

import com.gm.common.PageResult;
import com.gm.common.Result;
import com.gm.pojo.TbContent;

import java.util.List;

public interface ContentService {
    Result  findAll();

    PageResult findPage(Integer pageNo, Integer pageSize);

    Result findPage(TbContent content, Integer pageNo, Integer pageSize);

    Result saveContent(TbContent  content);

    Result updateContent(TbContent content);

    Result deleteContent(Long[] ids);

    Result deleteOneContent(Long id);
}
